src/shared/client/cg_api.cpp

```diff
@@ -326,8 +326,14 @@ void trap_R_AddPolyToScene( qhandle_t hShader, int numVerts, const polyVert_t *v
 
 void trap_R_AddPolysToScene( qhandle_t hShader, int numVerts, const polyVert_t *verts, int numPolys )
 {
-	std::vector<polyVert_t> myverts(numVerts * numPolys);
-	memcpy(myverts.data(), verts, numVerts * numPolys * sizeof(polyVert_t));
+	size_t size = (numVerts * numPolys);
+	if (!size) { return; }
+
+	/* Known to crash on Clang ≥ 14 in non-Debug native build:
```

Comment:
The comment needs more context to make sense--it doesn't say anything about the condition that something is 0. Also the spacing is arranged so as to imply it crashes in the std::vector constructor before getting to the IPC code?

Reply:
> The comment needs more context to make sense--it doesn't say anything about the condition that something is 0. 

Right, I will edit that.

> Also the spacing is arranged so as to imply it crashes in the std::vector constructor before getting to the IPC code?

According to this comment:

https://github.com/Unvanquished/Unvanquished/issues/2682#issuecomment-2080398702

It doesn't crash before calling `cmdBuffer.SendMsg()`. I can move the comment too.

Reply:
Done for both.